src/npe2/_inspection/_from_npe1.py

```diff
@@ -517,7 +517,13 @@ def get_top_module_path(package_name, top_module: Optional[str] = None) -> Path:
         top_module = top_mods[0]
 
     path = Path(dist.locate_file(top_module))
-    assert path.is_dir()
+    if not path.is_dir() and dist.files:
+        for f_path in dist.files:
+            if "__editable__" in f_path.name:
+                path = Path(f_path.read_text().strip()) / top_module
+                break
```

Comment:
@Czaki I have zero context here so take my review with a grain of salt, but if there are no tests here, could you at least add comments about why these lines are needed? eg "if the given npe1 plugin is installed in editable mode, then X doesn't work so we must do Y"